src/resources/views/create.blade.php

```diff
@@ -165,7 +165,7 @@ class="btn btn-success"
         </div>
 
         <p v-cloak v-if="exceedsMaxSize" class="text-danger">
-            Your upload is larger than the last <span v-text="remainingQuota"></span> of storage available to you.
+            You have selected more than the <span v-text="availableQuota"></span> of storage available to you.
```

Comment:
You can still show `remainingQuota`, as it is more helpful.
```suggestion
            You have selected more than the <span v-text="remainingQuota"></span> of storage available to you.
```